full_bubble_model.py

```diff
@@ -47,22 +47,34 @@
 from func_timeout import func_timeout, FunctionTimedOut   # for timeout
 import os    # file management
 import importlib   # for reloading your own files
+import traceback   # for error handling
 
-# my own files:
+# import parameters.py as par:
 try:
     import parameters as par   # numeric constants and coefficents
     importlib.reload(par)   # reload changes you made
-except:
+except Exception as _error:
     print(print(colored('Error, \'parameters.py\' not found','red')))
+    raise _error
+
+# import excitation.py as excitation:
 try:
     import excitation
     importlib.reload(excitation)
-except:
+except ImportError as _error:
```

Comment:
Better error handling, when importing:

So far, the role of this try-except block was to import either excitation or Bubble_synamics_simulation.excitation appropriately.

From now on, error messages won't be lost (e.g. numpy is not installed, syntax error in the file)

Reply:
same for all other imports